terraform/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "helm_release" "split-me-release" {
+  name       = "split-me-release"
+  repository = "https://rbalusup.github.io/Helm3"
+  chart      = "split-me-chart"
+  namespace = "default"
+}
```

Comment:
What's the process for deploying subsequent changes to the Python code or Helm chart?